plugins/backend/bazqux/bazquxAPI.vala

```diff
@@ -116,9 +116,9 @@ public class FeedReader.bazquxAPI : GLib.Object {
 			string feedID = object.get_string_member("id");
 			string url = object.has_member("htmlUrl") ? object.get_string_member("htmlUrl") : object.get_string_member("url");
 			string icon_url = object.has_member("iconUrl") ? object.get_string_member("iconUrl") : "";
-			if(icon_url != "" && !m_utils.downloadIcon(feedID, "https:"+icon_url))
+			if(icon_url != "" && !Utils.downloadIcon(feedID, "https:"+icon_url))
```

Comment:
I'm not the original author of this back-end. I just fixed up the code before merging. But I think the "iconUrl" field is actually missing the "https:"